src/ui/Dropdown.css

```diff
@@ -76,13 +76,19 @@
    default focus styles. This is better for accessibility. */
 @media screen and (-webkit-min-device-pixel-ratio:0) {
   .root select {
-    opacity: 1;
     color: transparent;
+    opacity: 1;
```

Comment:
Oh, that makes sense actually. I'm not particularly glued to the previous order, just like that the PR touch only the code it needs to fix.